server/models/index.js

```diff
@@ -84,6 +70,14 @@ const deleteSong = async (songId) => {
   return db.query(`DELETE FROM ${songsTable} WHERE id = $1`, [songId]);
 };
 
+ const addUser = async (data, usersTable = 'users') => {
```

Comment:
You no longer need to pass a table into the models from the controllers. I removed the testing logic from controllers. There's a check at the top of models/index.js that sets usersTable based on whether we're testing.